package.json

```diff
@@ -74,6 +74,27 @@
             "type": "object",
             "title": "Rust configuration",
             "properties": {
+                "rls.client.showStderrInOutputChannel": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Specifies whether RLS stderr will be shown in the output channel along with protocol logs. Requires reloading extension after change."
+                },
+                "rls.client.logToFile": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "When set to true, RLS stderr is logged to a file at workspace root level. Requires reloading extension after change."
+                },
+                "rls.client.revealOutputChannelOn": {
+                    "type":"string",
+                    "enum": [
+                        "info",
+                        "warn",
+                        "error",
+                        "never"
+                    ],
+                    "default": "error",
```

Comment:
Set to `error` because it's the default level when not nothing is specified in the `ClientOptions` for the vscode-languageclient, but we can stick with `never` ourselves